pytket/extensions/qiskit/backends/ibm_utils.py

```diff
@@ -70,3 +77,78 @@ def _batch_circuits(
         for n, indices in itertools.groupby(order, key=lambda i: n_shots[i])
     ]
     return batches, batch_order
+
+
+def _architecture_to_couplingmap(architecture: Architecture) -> CouplingMap:
+    """
+    Converts a pytket Architecture object to a Qiskit CouplingMap object.
+
+    :param architecture: Architecture to be converted
```

Comment:
Could you add some more details about the return value here?

Reply:
done